CONTRIBUTING.md

```diff
@@ -1,23 +1,78 @@
-# Contributing
+# Contributing Guidelines
 
-Thank you for considering and taking the time to contribute! Product focuses on 100% education as well as upskilling developing countries and rural areas.
+🎉 First of all, thanks for taking the time to contribute! 🎉
 
-The following are guidelines for contributing to this project.
+The following is a set of guidelines for contributing to this project. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
 
-## How to Report Bugs
+## Code of Conduct
 
-Please open [a new issue in the appropriate GitHub repository][new-issue] with steps to reproduce the problem you're experiencing.
+This project and everyone participating in it is governed by a [Code of Conduct](https://github.com/kaiwalyakoparkar/Lilliputian-Bot/blob/main/code_of_conduct.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [kaiwalyakoparkar@gmail.com](mailto:kaiwalyakoparkar@gmail.com).

```

Comment:
Hey, I would like if you add my discord server link here as well as an option to report. Here it is: https://discord.com/invite/jvdcY2NkXa